Documentation/gettingstarted/cilium_install.rst

```diff
@@ -9,81 +9,15 @@ Cilium pod will run in the ``kube-system`` namespace along with all
 other system relevant daemons and services.  The Cilium pod will run
 both the Cilium agent and the Cilium CNI plugin.
 
-To deploy Cilium, run:
+Choose the installation instructions for the environment that you are deploying
```

Comment:
"for the environment in which you are deploying Cilium."

Reply:
Done